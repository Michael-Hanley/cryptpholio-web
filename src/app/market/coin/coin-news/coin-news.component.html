<mat-list-item>
    <a href="{{url}}">
        <div class="list-item">
            <span><img class="article-image" src="{{imageUrl}}" /></span>
            <span class="article-title">{{title}}</span>
        </div>
    </a>
</mat-list-item>
<mat-divider></mat-divider>