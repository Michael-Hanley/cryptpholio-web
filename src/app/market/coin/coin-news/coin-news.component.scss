.article-image {
    margin: 5px 10px;
    max-width: 50px;
}
.article-title {
    margin-left: 10px;
}
