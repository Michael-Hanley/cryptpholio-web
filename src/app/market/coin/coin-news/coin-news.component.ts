import { Component, OnInit, Input } from '@angular/core';

@Component({
  selector: 'app-coin-news',
  templateUrl: './coin-news.component.html',
  styleUrls: ['./coin-news.component.scss']
})
export class CoinNewsComponent implements OnInit {
  @Input() title;
  @Input() imageUrl;
  @Input() url;
  constructor() { }

  ngOnInit() {
  }

}
