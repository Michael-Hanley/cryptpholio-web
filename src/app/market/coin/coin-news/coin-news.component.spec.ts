import { async, ComponentFixture, TestBed } from '@angular/core/testing';
import { MatListModule } from '@angular/material';
import { CoinNewsComponent } from './coin-news.component';

describe('CoinNewsComponent', () => {
  let component: CoinNewsComponent;
  let fixture: ComponentFixture<CoinNewsComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [CoinNewsComponent],
      imports: [
        MatListModule
      ]
    })
      .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(CoinNewsComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });
  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
