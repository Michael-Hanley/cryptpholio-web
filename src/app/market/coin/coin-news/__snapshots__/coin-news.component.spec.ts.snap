// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoinNewsComponent should match snapshot 1`] = `
<app-coin-news>
  <mat-list-item
    class="mat-list-item"
  >
    <div
      class="mat-list-item-content"
    >
      <div
        class="mat-list-item-ripple mat-ripple"
        mat-ripple=""
      />
      <div
        class="mat-list-text"
      />
      <a
        href=""
      >
        <div
          class="list-item"
        >
          <span>
            <img
              class="article-image"
              src=""
            />
          </span>
          <span
            class="article-title"
          >
            
          </span>
        </div>
      </a>
    </div>
  </mat-list-item><mat-divider
    aria-orientation="horizontal"
    class="mat-divider mat-divider-horizontal"
    role="separator"
  />
</app-coin-news>
`;
